pkg/operator/config/config.go

```diff
@@ -17,10 +17,11 @@ limitations under the License.
 package config
 
 const (
-	DefaultExternalDNSImage  = "docker.io/bitnami/external-dns:latest"
-	DefaultMetricsAddr       = "127.0.0.1:8080"
-	DefaultOperatorNamespace = "externaldns-operator"
-	DefaultOperandNamespace  = "externaldns"
+	DefaultExternalDNSImage     = "docker.io/bitnami/external-dns:latest"
+	DefaultMetricsAddr          = "127.0.0.1:8080"
+	DefaultOperatorNamespace    = "openshift-external-dns-operator"
+	DefaultOperandNamespace     = "external-dns"
```

Comment:
The goal being to make this upstreamable, would it make more sense for the default namespace not to include "openshift-", and to include a set of manifests or patches to configure the operator and operand to use "openshift-"-prefixed namespaces?

Reply:
Hm, that sounds familiar. https://github.com/openshift/external-dns-operator/pull/19#discussion_r682107340

Reply:
Oops!  Hm, I guess we'll need to de-OpenShift-ize some other things anyway, so I guess using "openshift-" is fine.  However, if the operator namespace has the "openshift-" prefix, so should the operand namespace.

Reply:
I'm fine to revert all back to `external-dns-operator` namespace.

Reply:
It's now `external-dns-operator`, will have to do something for OpenShift intergrations in the future.